components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java

```diff
@@ -80,6 +80,9 @@
     
     private final static Pattern LAST_INT_PATTERN = Pattern.compile("[^0-9]+([0-9]+)$");
 
+    /*  Define property name to get member_id*/
+    private static final String memberIDKey = "member_id";
```

Comment:
We already have a constant for "member_id": LoadBalancerConstants.MEMBER_ID. Would you mind removing this and use the existing?
Thanks